myapp/db/migrate/20230106075124_add_tasks_count_to_users.rb

```diff
@@ -0,0 +1,18 @@
+class AddTasksCountToUsers < ActiveRecord::Migration[6.0]
+  def up
+    add_column :users, :tasks_count, :integer, default: 0, null: false
+
+    reversible do |dir|
+      dir.up { data }
+    end
+  end
+
+  def data
+    execute <<-SQL.squish
+        UPDATE users
+           SET tasks_count = (SELECT count(1)
+                              FROM tasks
+                              WHERE tasks.user_id = users.id)
+    SQL
```

Comment:
ref: [Proper Counter Cache Migrations in Rails](http://ryan.mcgeary.org/2016/02/05/proper-counter-cache-migrations-in-rails/)